go.mod

```diff
@@ -6,7 +6,10 @@ go 1.17
 exclude k8s.io/kubernetes v1.13.0
 
 // Client-go does not handle different versions of mergo due to some breaking changes - use the matching version
-replace github.com/imdario/mergo => github.com/imdario/mergo v0.3.5
+replace (
+	github.com/imdario/mergo => github.com/imdario/mergo v0.3.5
+	istio.io/api v1.13.3 => github.com/jewertow/api v1.13.3
```

Comment:
This is only a temporary change and I will revert it once my pull request to the Istio API is merged.